pyproject.toml

```
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of base32-lib
+# Copyright (C) 2019 CERN.
+# Copyright (C) 2019 Northwestern University,
+#                    Galter Health Sciences Library & Learning Center.
+# Copyright (C) 2025 Front Matter.
+
+# base32-lib is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+[build-system]
+requires = ["setuptools>=45", "wheel", "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "base32-lib"
+dynamic = ["version"]
+description = "Small library to generate, encode and decode random base32 strings."
+readme = { file = "README.rst", content-type = "text/x-rst" }
+requires-python = ">=3.8"
+license = { text = "MIT License" }
+authors = [{ name = "Invenio Software", email = "info@inveniosoftware.org" }]
+keywords = ["base32", "identifiers"]
+classifiers = [
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+]
+
+[project.urls]
+Homepage = "https://github.com/inveniosoftware/base32-lib"
+
+[project.optional-dependencies]
+docs = ["Sphinx>=1.4.2"]
+tests = [
+  "check-manifest>=0.25",
+  "coverage>=4.0",
+  "isort>=4.2.2",
+  "pydocstyle>=1.0",
+  "pytest-cov>=1.8.0",
+  "pytest>=3.6.0",
+]
```

I'd bump those to more modern lower bounds (for `tests` I think that's fair) akin to https://github.com/inveniosoftware/pytest-invenio/blob/master/setup.cfg#L29 but even more up-to-date:

```suggestion
tests = [
  "check-manifest>=0.49",
  "pytest>=6.0,<9.0",
  "pytest-cov>=3.0.0",
  "pytest-black-ng>0.4.0",
  "pytest-isort>=3.0.0",
  "pytest-pycodestyle>=2.2.0",
  "pytest-pydocstyle>=2.2.3",
]
```